SPECS/kernel/config

```diff
@@ -115,20 +119,21 @@ CONFIG_BPF_JIT=y
 CONFIG_BPF_JIT_ALWAYS_ON=y
 CONFIG_BPF_JIT_DEFAULT_ON=y
 CONFIG_BPF_UNPRIV_DEFAULT_OFF=y
+CONFIG_USERMODE_DRIVER=y
 # CONFIG_BPF_PRELOAD is not set
-# CONFIG_BPF_LSM is not set
+CONFIG_BPF_LSM=y
```

Comment:
Enables instrumentation of the security hooks with BPF programs for implementing dynamic MAC and Audit Policies.

https://cateee.net/lkddb/web-lkddb/BPF_LSM.html